sway/commands/seat/cursor.c

```diff
@@ -32,6 +33,7 @@ static struct cmd_results *handle_command(struct sway_cursor *cursor,
 		float x = strtof(argv[1], NULL) / root->width;
 		float y = strtof(argv[2], NULL) / root->height;
 		wlr_cursor_warp_absolute(cursor->cursor, NULL, x, y);
+		wlr_seat_pointer_notify_frame(cursor->seat->wlr_seat);
 		cursor_rebase(cursor);
```

Comment:
Hmm, perhaps this might be better placed in `cursor_rebase`, or at least come after the call to it since it's the one that's sending enter/motion events here, not `wlr_cursor_warp_absolute`.

Reply:
Sorry, meant in the seatop implementing `cursor_rebase`.

Reply:
I moved it to after `cursor_rebase`. I don't think emitting it inside `cursor_rebase` (or the seatop handler) is a good idea since we call `cursor_rebase` in places where there is no cursor event - such as when a view gets resized or unmapped.

Reply:
That sounds good then. I was concerned about whether we sent a frame to the surface underneath a floating window when the floating window is unmapped while having pointer focus, but it looks like we do anyway.